pyiron_base/jobs/job/runfunction.py

```diff
@@ -456,6 +454,19 @@ def run_job_with_runmode_srun(job):
     )
 
 
+def run_static_decorator(func):
+    def wrapper(job):
+        if job.job_id is not None:
+            job.project.db.item_update({"timestart": datetime.now()}, job.job_id)
```

Comment:
Based on [my comment](https://github.com/pyiron/pyiron_base/issues/979#issuecomment-1384891292), I would suggest to write:

```python
job.project.db.item_update({"timestart": datetime.now(), "timestop": None, "totalcputime": None}, job.job_id)
```

Reply:
> Based on [my comment](https://github.com/pyiron/pyiron_base/issues/979#issuecomment-1384891292), I would suggest to write:
> 
> ```python
> update_dict = {"timestop": None, "totalcputime": None}
> if job.project.db.get_item_by_id(job._job_id)["timestart"] is None:
>     update_dict["timestart"] = datetime.now()
> job.project.db.item_update(update_dict, job.job_id)
> ```

Running `run_static()` multiple times currently only works for the `GenericMasters` including the `InteractiveWrapper`. For a regular job this is something we do not support at the moment, for example for VASP we would overwrite the output, so to reduce the overhead I would not add another database request to get the `timestart` currently stored in the database.